Library/Formula/snappystream.rb

```diff
@@ -14,9 +14,14 @@ class Snappystream < Formula
 
   depends_on "cmake" => :build
   depends_on "snappy"
+  depends_on "boost" => :optional
 
   def install
-    system "cmake", ".", "-DBUILD_TESTS=ON", *std_cmake_args
+    args = [ ".", "-DBUILD_TESTS=ON" ]
+    if build.with? "boost"
+      args << "-DWITH_BOOST_IOSTREAMS=1"
+    end
+    system "cmake", *args, *std_cmake_args
```

Comment:
Thank you, totally forgot about asterisk expansion. Changed suggested code a little.